cinema/serializers.py

```diff
@@ -59,6 +68,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The `tickets_available` field is defined as a read-only `IntegerField`, but it does not specify a source or method to calculate its value. You have defined a `get_tickets_available` method, but it is not being used. To fix this, you should use a `SerializerMethodField` for `tickets_available` and link it to the `get_tickets_available` method.